tests/Platforms/OraclePlatformTest.php

```diff
@@ -829,6 +830,72 @@ public function testQuotedTableNames(): void
         self::assertEquals($createTriggerStatement, $sql[3]);
     }
 
+    public function testDropIndexWithoutQuotedIndexName(): void
+    {
+        $sql = $this->platform->getDropIndexSQL('IDX_NAME');
+        self::assertEquals('DROP INDEX IDX_NAME', $sql);
+    }
+
+    public function testDropIndexWithInitiallyQuotedIndexName(): void
+    {
+        $sql = $this->platform->getDropIndexSQL('idx_name');
```

Comment:
The body of your PR describes initially quoted things as "the_thing". Here, we have `'idx_name'`, shouldn't it be `'"idx_name"'`? Sorry if it's a dumb question, but I know very little about Oracle, and nothing seems quoted to me here.

Reply:
That's the whole point. The index has been created initially using:
```
CREATE INDEX "idx_name" ...
```
The schema manager returns unquoted `idx_name` from the database and passes it to `getDropIndexSQL()` without quotes and the method doesn't know any more that it was quoted in the initial `CREATE INDEX` statement. Before the PR, it has been sending this to the database:
```
DROP INDEX IDX_NAME
```
which fails because `IDX_NAME` is no known index but `"idx_name"` is. Therefore, we now check if the name is all upper case and if not, the name must have been quoted in the initial `CREATE INDEX` statement.

Reply:
Okay, I don't know if it's the best solution, but at least it makes sense now! Thanks!